PROJECT

```diff
@@ -0,0 +1,19 @@
+# Code generated by tool. DO NOT EDIT.
+# This file is used to track the info used to scaffold your project
+# and allow the plugins properly work.
+# More info: https://book.kubebuilder.io/reference/project-config.html
+layout:
+- ansible.sdk.operatorframework.io/v1
+multigroup: true
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+projectName: nri-plugins-operator
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  group: config.nri
+  kind: NriPluginDeployment
```

Comment:
nitpick: since NRI is a TLA, should this be `NRIPluginDeployment` ?

Reply:
as discussed during the call, let's keep it for now as it is and we can revisit it if we find some best practices that suggests using capitalized letters for the abbreviations.